deploy/playbooks/roles/common/templates/prod_nodes.py.j2

```diff
@@ -162,7 +142,7 @@ nodes = {
         'keyname': keyname,
         'image_name': 'Centos 7',
         'size': 'hg-30-flex',
-        'labels': ['vagrant', 'ceph-ansible'],
+        'labels': ['ovh', 'vagrant', 'ceph-ansible'],
```

Comment:
this needs a ``libvirt`` label.

Actually, do we even need this and a xenial one?